lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -360,6 +398,7 @@ testRule({
 	ruleName,
 	config: [true],
 	customSyntax: 'postcss-css-in-js',
+	fix: true,
```

Comment:
Ah, sorry. The cases for `postcss-css-in-js` has been skipped, so, please revert the change in this `testRule`?